react-props-and-expressions/my-app/src/CustomButton.js

```diff
@@ -0,0 +1,7 @@
+import './CustomButton.css';
+
+export default function CustomButton({ buttons }) {
+  return buttons.map((button) => {
```

Comment:
Per my comment in App.js, if you had three separate instances of your CustomButton component this code would be handled differently. 

However, with the approach you took, this is a fine implementation in the component.